cms/migrations/0027_populate_slug_values_faqs.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2016-12-21 19:26
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from django.utils.text import slugify
+
+
+def gen_unique_slug(apps, schema_editor):
+    FrequentlyAskedQuestion = apps.get_model('cms', 'FrequentlyAskedQuestion')
+    for row in FrequentlyAskedQuestion.objects.all():
+        if not row.slug:
+            slug = orig_slug = slugify(row.question)
+            slug_is_unique = not FrequentlyAskedQuestion.objects.filter(slug=orig_slug).exists()
+            count = 1
+            while not slug_is_unique:
+                slug = "{orig}-{count}".format(orig=orig_slug, count=count)
+                slug_is_unique = not FrequentlyAskedQuestion.objects.filter(slug=slug).exists()
+                count += 1
+            row.slug = slug
+        row.save()
```

Comment:
This line should be indented by one. There's no need to save _every_ row, only the rows that didn't previously have slugs.